gradle.properties

```diff
@@ -23,3 +23,47 @@ android.useAndroidX=true
 android.enableJetifier=false
 
 kotlin.parallel.tasks.in.project=true
+
+splitties.version=3.0.0-dev-035
+
+# Dependencies and Plugin versions with their available updates
+# Generated by $ ./gradlew refreshVersions
+# You can edit the rest of the file, it will be kept intact
+# See https://github.com/jmfayard/buildSrcVersions/issues/77
+plugin.com.github.ben-manes.versions=0.25.0
+plugin.de.fayard.buildSrcVersions=0.6.2
+version.androidx.test..core-ktx=1.1.0
+#                   # available=1.2.0
+version.androidx.test..runner=1.1.1
+#                 # available=1.2.0
+version.org.jetbrains.kotlinx=1.3.0
```

Comment:
Should be kotlinx.coroutines. Non coroutines kotlinx artifacts have different versions. Can you add kotlinx.serialization with its current version to ensure it works correctly?

Reply:
Good point, I did this in the newer PR